common/authorization/authorizer.go

```diff
@@ -50,6 +50,8 @@ type CallTarget struct {
 	APIName string
 	// If a Namespace is not being targeted this be set to an empty string.
 	Namespace string
+	// If a Namespace is not being targeted this be set to an empty string.
+	Request interface{}
```

Comment:
if the PR aims to block cross namespace call, then what is this ?

Reply:
It is some leftover but actually there was a user request to add `Request` to `CallTarget`.